praw/models/reddit/subreddit.py

```diff
@@ -1586,6 +1586,41 @@ def conversations(self, after=None, limit=None, other_subreddits=None,
             yield ModmailConversation.parse(data, self.subreddit._reddit,
                                             convert_objects=False)
 
+    def create(self, subject, body, recipient, author_hidden=False):
+        """Create a new modmail conversation.
+
+        :param subject: The message subject. Cannot be empty.
```

Comment:
For subject, probably not. For the message body, I was thinking on the model of email and Reddit self-posts, where blank messages are allowed; so, it might be nice to document the difference.